Topics/Development_Process.md

```diff
@@ -131,20 +131,31 @@ The SOLID principles are:
 ## Restaurant example of each principle
 
 #### SRP
-Consider a **Chef** in a restaurant. Initially, the chef might handle multiple responsibilities: preparing dishes, managing inventory, and overseeing kitchen hygiene. Adhering to SRP, distinct roles are established. The chef focuses solely on cooking, while inventory management becomes the responsibility of a dedicated inventory manager. 
+Consider a **Chef** in a restaurant. Initially, the chef might handle multiple responsibilities: preparing dishes, managing inventory, and overseeing kitchen hygiene. Adhering to SRP, distinct roles are established. The chef focuses solely on cooking and kitchen hygiene, while inventory management becomes the responsibility of a dedicated inventory manager. 
+
+![SRP](https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/90370938/92398ca4-dc69-40d5-bd42-34488c2aab05)
```

Comment:
Looks good, but I think it would help to add arrows to illustrate the depdencies between classes like you did for the Depdency Inversion Principle.